_posts/2019-01-25-lambda-to-openfaas-with-skill.md

```diff
@@ -0,0 +1,280 @@
+---
+title: "A snowman's journey: Lambda to OpenFaaS"
+description: Richard Gee showcases a holiday project which demonstrates adding voice control to OpenFaaS functions using an Alexa skill and moving the handling function out of AWS Lambda onto OpenFaaS alternatives.
+date: 2019-01-25
+image: /images/lambda-to-openfaas-with-skill/background-brown-christmas.jpg
+categories:
+  - alexa
+  - lambda
+  - cloud
+author_staff_member: richard
+dark_background: true
+---
+
+As the festive season started to approach, as it tends to on a fairly regular and annual basis, I decided to revisit a project from the previous year where I used Amazon Echo to control a festive decoration.  The _smart_ festive decoration remains synchronised with the colour of devices across the globe using the current [Cheerlights](http://cheerlights.com/) colour. 
```

Comment:
It might be considered a `nit`, but we don't use double spaces after sentences anywhere in the documentation, existing blogs or in markdown. Can you sed `  ` -> ` `?